pom.xml

```diff
@@ -151,49 +151,34 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.9.0</version>
-            <scope>compile</scope>
+            <version>2.10.1</version>
         </dependency>
 
         <!-- Apache Http Client -->
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.13</version>
-            <scope>compile</scope>
```

Comment:
Scope is compile by default so I decided to not include it to keep things less busy.